hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/security/RouterDelegationTokenSecretManager.java

```diff
@@ -63,8 +63,9 @@
    *                                           in milliseconds
    * @param delegationTokenRemoverScanInterval how often the tokens are scanned
    */
-  public RouterDelegationTokenSecretManager(long delegationKeyUpdateInterval, long delegationTokenMaxLifetime,
-      long delegationTokenRenewInterval, long delegationTokenRemoverScanInterval) {
+  public RouterDelegationTokenSecretManager(long delegationKeyUpdateInterval,
```

Comment:
Do we need to do this?

Reply:
RouterDelegationTokenSecretManager.java is a new file, the length of the constructor parameter exceeds 100 characters.

```
RouterDelegationTokenSecretManager.java:66:  public RouterDelegationTokenSecretManager(long 
delegationKeyUpdateInterval, long delegationTokenMaxLifetime,: Line is longer than 100 characters (found 110). 
[LineLength]
```